### PR TITLE
Add ansible 2.0.2

### PR DESCRIPTION
**Pull request by powersjcb**
Pulled from Homebrew/homebrew-core: af9c040d4ba13d6c023d29c4d7326ab722768a49

Ansible 2.1 is new default.

### CODE REVIEW

**Comment by ssbarnea** [comment]
Time to close this one.

File: ansible20.rb [hunk 1]
```diff
@@ -0,0 +1,609 @@
...
+  url "https://releases.ansible.com/ansible/ansible-2.0.2.0.tar.gz"
+  sha256 "373a2e50319d90da50948e3faf1c033464b7302200e0199da8981d24646d4387"
+
+  head "https://github.com/ansible/ansible.git", :branch => "devel"
+
+  bottle do
+    sha256 "b24e695649a6467d13288e892910b677eb16a5dcb197ecfe09839ff8c536115d" => :el_capitan
+    sha256 "71b8c4e609a97225dbf816ecb8430feb9f27649be9d64ac6045bb63671c01a1a" => :yosemite
+    sha256 "260b5ce5252d6071580e6e14a7147a2642b6fbf54fa7152daea15a0cfddf936b" => :mavericks
+  end
```
Comment (MikeMcQuaid) [inline ansible20.rb]: Please remove this block.

**Comment by MikeMcQuaid** [comment]
Please also remove `ansible14.rb` and `ansible18.rb` in this PR.

File: ansible20.rb [hunk 2]
```diff
@@ -0,0 +1,609 @@
+class Ansible20 < Formula
+  desc "Automate deployment, configuration, and upgrading"
+  homepage "https://www.ansible.com/"
+  url "https://releases.ansible.com/ansible/ansible-2.0.2.0.tar.gz"
+  sha256 "373a2e50319d90da50948e3faf1c033464b7302200e0199da8981d24646d4387"
+
+  head "https://github.com/ansible/ansible.git", :branch => "devel"
```
Comment (DomT4) [inline ansible20.rb]: Versions formulae don't have `head`s unless those are pegged to the same release branch as the version.

**Comment by DomT4** [comment]
This will either need to `conflicts_with "ansible", :because => "blah"` or be `keg_only`. I don't mind which.

**Comment by powersjcb** [comment]
Comments have been addressed in latest commit. Thanks for the fast feedback.

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `ansible20 2.0.2 (new formula)`. Thanks!

**Pull request closed without merging**